### PR TITLE
Docs/goals-intro

### PR DESCRIPTION
**Pull request by mighdoll**
add goals and target programmer section to readme

### CODE REVIEW

**Review by k2d222 — COMMENTED** [review]
Same comment as the other PR, it's fine here for now but when we have our shiny specs and impls to showcase, I would move these discussions to another file, for advanced users to read.

Our frontpage should quickly expose what we do, with links to more details.

File: README.md [hunk 1]
```diff
@@ -31,15 +31,47 @@ A standard community library format is key to enable an WGSL library ecosystem.
...
+The target project size for enhanced WGSL ranges from tiny programs
+up to teams of few dozen programmers.
+Software engineering features useful for very large teams or very large
+projects are unlikely to be needed for enhanced WGSL.
+
 ## WGSL Enhancements
 
 We're aiming to prioritize WGSL enhancements that are: 1) important for community projects,
 2) feel natural to the WGSL programmer,
 and 3) reasonable to integrate into tools.
```
Comment (stefnotch) [inline README.md:66]: How about

"and 4) are simple to implement, helping widespread adoption"

Comment (mighdoll) [reply README.md:66]: maybe replace 3) with 'simple to integrate'? (having both 'reasonable to integrate' and 'simple to integrate')

**Comment by mighdoll** [comment]
> Same comment as the other PR, it's fine here for now but when we have our shiny specs and impls to showcase, I would move these discussions to another file, for advanced users to read.
> 
> Our frontpage should quickly expose what we do, with links to more details.

I agree, doesn't belong here. will move to another file.

File: README.md (hunk 1 above)
Comment (stefnotch) [reply README.md:66]: Sounds good

**Review by k2d222 — APPROVED** [review]

**Pull request merged**